spec/request/session/create_spec.rb

```diff
@@ -0,0 +1,59 @@
+require 'rails_helper'
+
+RSpec.describe 'POST #create sign_in', type: :request do
+  let!(:user_attrs) { { user: FactoryBot.attributes_for(:user) } }
+
+  subject do
+    post '/api/v1/users/sign_in', params: {
+      email: user_attrs[:user][:email], password: user_attrs[:user][:password]
+    }, as: :json
+  end
+
+  context 'when valid' do
+    context 'when registered and confirmed' do
+      let!(:user) { FactoryBot.create(:confirmed_user) }
```

Comment:
`FactoryBot` is not necessary, when you use the `create` here the tests will know that is FactoryBot